{"@t":"2022-02-22T07:46:27.5956281Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Booting Umbraco 8.17.2.","TimingId":"401f022","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":24244,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"de1e28a8-b099-4d5e-8e05-b51d2a34f3b5"}
{"@t":"2022-02-22T07:46:27.6286311Z","@mt":"Booting site '{HostingSiteName}', app '{HostingApplicationID}', path '{HostingPhysicalPath}', server '{MachineName}'.","HostingSiteName":"Flight2Orbit","HostingApplicationID":"/LM/W3SVC/2/ROOT","HostingPhysicalPath":"D:\\Users\\Oliver.Gjesing.EMEA\\Documents\\Git\\FTO\\BE\\Flight2Orbit\\","MachineName":"AAR00118","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":24244,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"de1e28a8-b099-4d5e-8e05-b51d2a34f3b5"}
{"@t":"2022-02-22T07:46:27.7566311Z","@mt":"Acquiring.","SourceContext":"Umbraco.Core.Runtime.MainDom","ProcessId":24244,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"de1e28a8-b099-4d5e-8e05-b51d2a34f3b5"}
{"@t":"2022-02-22T07:46:27.7586307Z","@mt":"Acquired.","SourceContext":"Umbraco.Core.Runtime.MainDom","ProcessId":24244,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"de1e28a8-b099-4d5e-8e05-b51d2a34f3b5"}
{"@t":"2022-02-22T07:46:28.7749758Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Resolving composer types.","TimingId":"9fcab51","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":24244,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"de1e28a8-b099-4d5e-8e05-b51d2a34f3b5"}
{"@t":"2022-02-22T07:46:28.9106434Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Resolved.","Duration":135,"TimingId":"9fcab51","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":24244,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"de1e28a8-b099-4d5e-8e05-b51d2a34f3b5"}
{"@t":"2022-02-22T07:46:29.5401809Z","@mt":"Profiler is VoidProfiler, not profiling (must run debug mode to profile).","SourceContext":"Umbraco.Web.Logging.WebProfilerComponent","ProcessId":24244,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"de1e28a8-b099-4d5e-8e05-b51d2a34f3b5"}
{"@t":"2022-02-22T07:46:29.7117840Z","@mt":"Initializing Umbraco internal event handlers for cache refreshing.","SourceContext":"Umbraco.Web.Cache.DistributedCacheBinderComponent","ProcessId":24244,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"de1e28a8-b099-4d5e-8e05-b51d2a34f3b5"}
{"@t":"2022-02-22T07:46:29.8047504Z","@mt":"Adding examine event handlers for {RegisteredIndexers} index providers.","RegisteredIndexers":3,"SourceContext":"Umbraco.Web.Search.ExamineComponent","ProcessId":24244,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"de1e28a8-b099-4d5e-8e05-b51d2a34f3b5"}
{"@t":"2022-02-22T07:46:29.8694359Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Booted.","Duration":2287,"TimingId":"401f022","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":24244,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"de1e28a8-b099-4d5e-8e05-b51d2a34f3b5"}
{"@t":"2022-02-22T07:46:30.3383253Z","@mt":"New url {Url} detected, re-discovering application url.","Url":"http://localhost:54252/umbraco","SourceContext":"Umbraco.Core.Sync.ApplicationUrlHelper","ProcessId":24244,"ProcessName":"iisexpress","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"3fbf8b89-b53c-4796-a918-c550039ec03f"}
{"@t":"2022-02-22T07:46:30.3413329Z","@mt":"ApplicationUrl: {UmbracoAppUrl} (UmbracoModule request)","UmbracoAppUrl":"http://localhost:54252/umbraco","SourceContext":"Umbraco.Core.Sync.ApplicationUrlHelper","ProcessId":24244,"ProcessName":"iisexpress","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"3fbf8b89-b53c-4796-a918-c550039ec03f"}
{"@t":"2022-02-22T07:46:30.9795958Z","@mt":"Registered with MainDom, localContentDbExists? {LocalContentDbExists}, localMediaDbExists? {LocalMediaDbExists}","LocalContentDbExists":true,"LocalMediaDbExists":true,"SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":24244,"ProcessName":"iisexpress","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"3fbf8b89-b53c-4796-a918-c550039ec03f"}
{"@t":"2022-02-22T07:46:30.9805966Z","@mt":"Creating the content store, localContentDbExists? {LocalContentDbExists}","LocalContentDbExists":true,"SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":24244,"ProcessName":"iisexpress","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"3fbf8b89-b53c-4796-a918-c550039ec03f"}
{"@t":"2022-02-22T07:46:30.9815956Z","@mt":"Creating the media store, localMediaDbExists? {LocalMediaDbExists}","LocalMediaDbExists":true,"SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":24244,"ProcessName":"iisexpress","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"3fbf8b89-b53c-4796-a918-c550039ec03f"}
{"@t":"2022-02-22T07:46:31.8122628Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Loading content from local cache file","TimingId":"5150d67","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":24244,"ProcessName":"iisexpress","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"3fbf8b89-b53c-4796-a918-c550039ec03f"}
{"@t":"2022-02-22T07:46:31.8482940Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Completed.","Duration":35,"TimingId":"5150d67","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":24244,"ProcessName":"iisexpress","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"3fbf8b89-b53c-4796-a918-c550039ec03f"}
{"@t":"2022-02-22T07:46:31.8612597Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Loading media from local cache file","TimingId":"4820123","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":24244,"ProcessName":"iisexpress","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"3fbf8b89-b53c-4796-a918-c550039ec03f"}
{"@t":"2022-02-22T07:46:31.8682939Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Completed.","Duration":7,"TimingId":"4820123","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":24244,"ProcessName":"iisexpress","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"3fbf8b89-b53c-4796-a918-c550039ec03f"}
{"@t":"2022-02-22T07:46:32.1035663Z","@mt":"Forcing index {IndexerName} to be unlocked since it was left in a locked state","IndexerName":"InternalIndex","SourceContext":"Umbraco.Examine.ExamineExtensions","ProcessId":24244,"ProcessName":"iisexpress","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"3fbf8b89-b53c-4796-a918-c550039ec03f"}
{"@t":"2022-02-22T07:46:32.1045328Z","@mt":"Forcing index {IndexerName} to be unlocked since it was left in a locked state","IndexerName":"MembersIndex","SourceContext":"Umbraco.Examine.ExamineExtensions","ProcessId":24244,"ProcessName":"iisexpress","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"3fbf8b89-b53c-4796-a918-c550039ec03f"}
{"@t":"2022-02-22T07:46:32.1045328Z","@mt":"Forcing index {IndexerName} to be unlocked since it was left in a locked state","IndexerName":"ExternalIndex","SourceContext":"Umbraco.Examine.ExamineExtensions","ProcessId":24244,"ProcessName":"iisexpress","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"3fbf8b89-b53c-4796-a918-c550039ec03f"}
{"@t":"2022-02-22T07:46:32.1065356Z","@mt":"Starting initialize async background thread.","SourceContext":"Umbraco.Web.Search.BackgroundIndexRebuilder","ProcessId":24244,"ProcessName":"iisexpress","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"3fbf8b89-b53c-4796-a918-c550039ec03f"}
{"@t":"2022-02-22T07:55:40.2757167Z","@mt":"[{EventType}] Event Id: {EventId}, State: {State}","EventType":"Information","EventId":0,"State":"Login attempt succeeded for username oliver.gjesing@akqa.com from IP address ::1","SourceContext":"Umbraco.Core.Logging.OwinLogger","ProcessId":24244,"ProcessName":"iisexpress","ThreadId":61,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":3,"HttpRequestId":"99fc215e-3308-417e-bab7-6eacc6f3cab9"}
{"@t":"2022-02-22T07:55:40.2767563Z","@mt":"[{EventType}] Event Id: {EventId}, State: {State}","EventType":"Information","EventId":0,"State":"User: oliver.gjesing@akqa.com logged in from IP address ::1","SourceContext":"Umbraco.Core.Logging.OwinLogger","ProcessId":24244,"ProcessName":"iisexpress","ThreadId":61,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":3,"HttpRequestId":"99fc215e-3308-417e-bab7-6eacc6f3cab9"}
{"@t":"2022-02-22T08:15:33.4023132Z","@mt":"[{EventType}] Event Id: {EventId}, State: {State}","EventType":"Information","EventId":0,"State":"User logged will be logged out due to timeout: oliver.gjesing@akqa.com, IP Address: ::1","SourceContext":"Umbraco.Core.Logging.OwinLogger","ProcessId":24244,"ProcessName":"iisexpress","ThreadId":225,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":4,"HttpRequestId":"a26febf0-dd3e-42a8-9472-1335a5af646a"}
{"@t":"2022-02-22T08:15:39.4194623Z","@mt":"[{EventType}] Event Id: {EventId}, State: {State}","EventType":"Information","EventId":0,"State":"User logged will be logged out due to timeout: oliver.gjesing@akqa.com, IP Address: ::1","SourceContext":"Umbraco.Core.Logging.OwinLogger","ProcessId":24244,"ProcessName":"iisexpress","ThreadId":238,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":5,"HttpRequestId":"ee70c7f7-2077-47cc-a9c2-8d6d12c68a84"}
{"@t":"2022-02-22T08:26:26.9754189Z","@mt":"Failed (will repeat).","@l":"Error","@x":"System.Data.SqlClient.SqlException (0x80131904): Execution Timeout Expired.  The timeout period elapsed prior to completion of the operation or the server is not responding. ---> System.ComponentModel.Win32Exception (0x80004005): The wait operation timed out\r\n   at System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)\r\n   at System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)\r\n   at System.Data.SqlClient.TdsParserStateObject.ReadSniError(TdsParserStateObject stateObj, UInt32 error)\r\n   at System.Data.SqlClient.TdsParserStateObject.ReadSniSyncOverAsync()\r\n   at System.Data.SqlClient.TdsParserStateObject.TryReadNetworkPacket()\r\n   at System.Data.SqlClient.TdsParserStateObject.TryPrepareBuffer()\r\n   at System.Data.SqlClient.TdsParserStateObject.TryReadByte(Byte& value)\r\n   at System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)\r\n   at System.Data.SqlClient.TdsParser.Run(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj)\r\n   at System.Data.SqlClient.TdsParser.TdsExecuteTransactionManagerRequest(Byte[] buffer, TransactionManagerRequestType request, String transactionName, TransactionManagerIsolationLevel isoLevel, Int32 timeout, SqlInternalTransaction transaction, TdsParserStateObject stateObj, Boolean isDelegateControlRequest)\r\n   at System.Data.SqlClient.SqlInternalConnectionTds.ExecuteTransactionYukon(TransactionRequest transactionRequest, String transactionName, IsolationLevel iso, SqlInternalTransaction internalTransaction, Boolean isDelegateControlRequest)\r\n   at System.Data.SqlClient.SqlInternalConnection.BeginSqlTransaction(IsolationLevel iso, String transactionName, Boolean shouldReconnect)\r\n   at System.Data.SqlClient.SqlConnection.BeginTransaction(IsolationLevel iso, String transactionName)\r\n   at System.Data.SqlClient.SqlConnection.BeginDbTransaction(IsolationLevel isolationLevel)\r\n   at StackExchange.Profiling.Data.ProfiledDbConnection.BeginDbTransaction(IsolationLevel isolationLevel) in C:\\projects\\dotnet\\src\\MiniProfiler.Shared\\Data\\ProfiledDbConnection.cs:line 138\r\n   at NPoco.Database.BeginTransaction(IsolationLevel isolationLevel)\r\n   at Umbraco.Core.Scoping.Scope.get_Database() in D:\\a\\1\\s\\src\\Umbraco.Core\\Scoping\\Scope.cs:line 330\r\n   at Umbraco.Core.Sync.DatabaseServerMessenger.Sync() in D:\\a\\1\\s\\src\\Umbraco.Core\\Sync\\DatabaseServerMessenger.cs:line 263\r\n   at Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent.InstructionProcessTask.PerformRun() in D:\\a\\1\\s\\src\\Umbraco.Web\\Compose\\DatabaseServerRegistrarAndMessengerComponent.cs:line 163\r\nClientConnectionId:c86a70a0-6e4b-4193-8ac8-a8cc005beacd\r\nError Number:-2,State:0,Class:11","SourceContext":"Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent+InstructionProcessTask","ProcessId":24244,"ProcessName":"iisexpress","ThreadId":303,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"ERROR"}
{"@t":"2022-02-22T08:26:48.6464162Z","@mt":"Failed (will repeat).","@l":"Error","@x":"System.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server) ---> System.ComponentModel.Win32Exception (0x80004005): The network path was not found\r\n   at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)\r\n   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)\r\n   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)\r\n   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)\r\n   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.Open()\r\n   at NPoco.Database.OpenSharedConnectionImp(Boolean isInternal)\r\n   at NPoco.Database.BeginTransaction(IsolationLevel isolationLevel)\r\n   at Umbraco.Core.Scoping.Scope.get_Database() in D:\\a\\1\\s\\src\\Umbraco.Core\\Scoping\\Scope.cs:line 330\r\n   at Umbraco.Core.Sync.DatabaseServerMessenger.Sync() in D:\\a\\1\\s\\src\\Umbraco.Core\\Sync\\DatabaseServerMessenger.cs:line 263\r\n   at Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent.InstructionProcessTask.PerformRun() in D:\\a\\1\\s\\src\\Umbraco.Web\\Compose\\DatabaseServerRegistrarAndMessengerComponent.cs:line 163\r\nClientConnectionId:00000000-0000-0000-0000-000000000000\r\nError Number:53,State:0,Class:20","SourceContext":"Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent+InstructionProcessTask","ProcessId":24244,"ProcessName":"iisexpress","ThreadId":309,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"ERROR"}
{"@t":"2022-02-22T08:26:49.9344537Z","@mt":"Failed to update server record in database.","@l":"Error","@x":"System.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server) ---> System.ComponentModel.Win32Exception (0x80004005): The network path was not found\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)\r\n   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.Open()\r\n   at NPoco.Database.OpenSharedConnectionImp(Boolean isInternal)\r\n   at NPoco.Database.BeginTransaction(IsolationLevel isolationLevel)\r\n   at Umbraco.Core.Scoping.Scope.get_Database() in D:\\a\\1\\s\\src\\Umbraco.Core\\Scoping\\Scope.cs:line 330\r\n   at Umbraco.Core.Persistence.Repositories.Implement.ServerRegistrationRepository.PerformGetAll(Int32[] ids) in D:\\a\\1\\s\\src\\Umbraco.Core\\Persistence\\Repositories\\Implement\\ServerRegistrationRepository.cs:line 59\r\n   at Umbraco.Core.Cache.NoCacheRepositoryCachePolicy`2.GetAll(TId[] ids, Func`2 performGetAll) in D:\\a\\1\\s\\src\\Umbraco.Core\\Cache\\NoCacheRepositoryCachePolicy.cs:line 47\r\n   at Umbraco.Core.Services.Implement.ServerRegistrationService.TouchServer(String serverAddress, String serverIdentity, TimeSpan staleTimeout) in D:\\a\\1\\s\\src\\Umbraco.Core\\Services\\Implement\\ServerRegistrationService.cs:line 54\r\n   at Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent.TouchServerTask.PerformRun() in D:\\a\\1\\s\\src\\Umbraco.Web\\Compose\\DatabaseServerRegistrarAndMessengerComponent.cs:line 206\r\nClientConnectionId:00000000-0000-0000-0000-000000000000\r\nError Number:53,State:0,Class:20","SourceContext":"Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent","ProcessId":24244,"ProcessName":"iisexpress","ThreadId":317,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"ERROR"}
{"@t":"2022-02-22T08:26:53.6594601Z","@mt":"Failed (will repeat).","@l":"Error","@x":"System.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server) ---> System.ComponentModel.Win32Exception (0x80004005): The network path was not found\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)\r\n   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.Open()\r\n   at NPoco.Database.OpenSharedConnectionImp(Boolean isInternal)\r\n   at NPoco.Database.BeginTransaction(IsolationLevel isolationLevel)\r\n   at Umbraco.Core.Scoping.Scope.get_Database() in D:\\a\\1\\s\\src\\Umbraco.Core\\Scoping\\Scope.cs:line 330\r\n   at Umbraco.Core.Sync.DatabaseServerMessenger.Sync() in D:\\a\\1\\s\\src\\Umbraco.Core\\Sync\\DatabaseServerMessenger.cs:line 263\r\n   at Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent.InstructionProcessTask.PerformRun() in D:\\a\\1\\s\\src\\Umbraco.Web\\Compose\\DatabaseServerRegistrarAndMessengerComponent.cs:line 163\r\nClientConnectionId:00000000-0000-0000-0000-000000000000\r\nError Number:53,State:0,Class:20","SourceContext":"Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent+InstructionProcessTask","ProcessId":24244,"ProcessName":"iisexpress","ThreadId":308,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"ERROR"}
{"@t":"2022-02-22T08:27:14.3284221Z","@mt":"Failed (will repeat).","@l":"Error","@x":"System.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server) ---> System.ComponentModel.Win32Exception (0x80004005): The network path was not found\r\n   at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)\r\n   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)\r\n   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)\r\n   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)\r\n   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.Open()\r\n   at NPoco.Database.OpenSharedConnectionImp(Boolean isInternal)\r\n   at NPoco.Database.BeginTransaction(IsolationLevel isolationLevel)\r\n   at Umbraco.Core.Scoping.Scope.get_Database() in D:\\a\\1\\s\\src\\Umbraco.Core\\Scoping\\Scope.cs:line 330\r\n   at Umbraco.Core.Sync.DatabaseServerMessenger.Sync() in D:\\a\\1\\s\\src\\Umbraco.Core\\Sync\\DatabaseServerMessenger.cs:line 263\r\n   at Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent.InstructionProcessTask.PerformRun() in D:\\a\\1\\s\\src\\Umbraco.Web\\Compose\\DatabaseServerRegistrarAndMessengerComponent.cs:line 163\r\nClientConnectionId:00000000-0000-0000-0000-000000000000\r\nError Number:53,State:0,Class:20","SourceContext":"Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent+InstructionProcessTask","ProcessId":24244,"ProcessName":"iisexpress","ThreadId":315,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"ERROR"}
{"@t":"2022-02-22T08:27:34.9909880Z","@mt":"Failed (will repeat).","@l":"Error","@x":"System.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server) ---> System.ComponentModel.Win32Exception (0x80004005): The network path was not found\r\n   at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)\r\n   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)\r\n   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)\r\n   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)\r\n   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.Open()\r\n   at NPoco.Database.OpenSharedConnectionImp(Boolean isInternal)\r\n   at NPoco.Database.BeginTransaction(IsolationLevel isolationLevel)\r\n   at Umbraco.Core.Scoping.Scope.get_Database() in D:\\a\\1\\s\\src\\Umbraco.Core\\Scoping\\Scope.cs:line 330\r\n   at Umbraco.Core.Sync.DatabaseServerMessenger.Sync() in D:\\a\\1\\s\\src\\Umbraco.Core\\Sync\\DatabaseServerMessenger.cs:line 263\r\n   at Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent.InstructionProcessTask.PerformRun() in D:\\a\\1\\s\\src\\Umbraco.Web\\Compose\\DatabaseServerRegistrarAndMessengerComponent.cs:line 163\r\nClientConnectionId:00000000-0000-0000-0000-000000000000\r\nError Number:53,State:0,Class:20","SourceContext":"Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent+InstructionProcessTask","ProcessId":24244,"ProcessName":"iisexpress","ThreadId":246,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"ERROR"}
{"@t":"2022-02-22T08:27:54.6199812Z","@mt":"Failed (will repeat).","@l":"Error","@x":"System.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server) ---> System.ComponentModel.Win32Exception (0x80004005): The network path was not found\r\n   at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)\r\n   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)\r\n   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)\r\n   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)\r\n   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.Open()\r\n   at NPoco.Database.OpenSharedConnectionImp(Boolean isInternal)\r\n   at NPoco.Database.BeginTransaction(IsolationLevel isolationLevel)\r\n   at Umbraco.Core.Scoping.Scope.get_Database() in D:\\a\\1\\s\\src\\Umbraco.Core\\Scoping\\Scope.cs:line 330\r\n   at Umbraco.Core.Sync.DatabaseServerMessenger.Sync() in D:\\a\\1\\s\\src\\Umbraco.Core\\Sync\\DatabaseServerMessenger.cs:line 263\r\n   at Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent.InstructionProcessTask.PerformRun() in D:\\a\\1\\s\\src\\Umbraco.Web\\Compose\\DatabaseServerRegistrarAndMessengerComponent.cs:line 163\r\nClientConnectionId:00000000-0000-0000-0000-000000000000\r\nError Number:53,State:0,Class:20","SourceContext":"Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent+InstructionProcessTask","ProcessId":24244,"ProcessName":"iisexpress","ThreadId":285,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"ERROR"}
{"@t":"2022-02-22T08:28:25.4237578Z","@mt":"Failed (will repeat).","@l":"Error","@x":"System.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server) ---> System.ComponentModel.Win32Exception (0x80004005): The network path was not found\r\n   at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)\r\n   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)\r\n   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)\r\n   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)\r\n   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.Open()\r\n   at NPoco.Database.OpenSharedConnectionImp(Boolean isInternal)\r\n   at NPoco.Database.BeginTransaction(IsolationLevel isolationLevel)\r\n   at Umbraco.Core.Scoping.Scope.get_Database() in D:\\a\\1\\s\\src\\Umbraco.Core\\Scoping\\Scope.cs:line 330\r\n   at Umbraco.Core.Sync.DatabaseServerMessenger.Sync() in D:\\a\\1\\s\\src\\Umbraco.Core\\Sync\\DatabaseServerMessenger.cs:line 263\r\n   at Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent.InstructionProcessTask.PerformRun() in D:\\a\\1\\s\\src\\Umbraco.Web\\Compose\\DatabaseServerRegistrarAndMessengerComponent.cs:line 163\r\nClientConnectionId:00000000-0000-0000-0000-000000000000\r\nError Number:53,State:0,Class:20","SourceContext":"Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent+InstructionProcessTask","ProcessId":24244,"ProcessName":"iisexpress","ThreadId":315,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"ERROR"}
{"@t":"2022-02-22T08:28:30.4337536Z","@mt":"Failed (will repeat).","@l":"Error","@x":"System.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server) ---> System.ComponentModel.Win32Exception (0x80004005): The network path was not found\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)\r\n   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.Open()\r\n   at NPoco.Database.OpenSharedConnectionImp(Boolean isInternal)\r\n   at NPoco.Database.BeginTransaction(IsolationLevel isolationLevel)\r\n   at Umbraco.Core.Scoping.Scope.get_Database() in D:\\a\\1\\s\\src\\Umbraco.Core\\Scoping\\Scope.cs:line 330\r\n   at Umbraco.Core.Sync.DatabaseServerMessenger.Sync() in D:\\a\\1\\s\\src\\Umbraco.Core\\Sync\\DatabaseServerMessenger.cs:line 263\r\n   at Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent.InstructionProcessTask.PerformRun() in D:\\a\\1\\s\\src\\Umbraco.Web\\Compose\\DatabaseServerRegistrarAndMessengerComponent.cs:line 163\r\nClientConnectionId:00000000-0000-0000-0000-000000000000\r\nError Number:53,State:0,Class:20","SourceContext":"Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent+InstructionProcessTask","ProcessId":24244,"ProcessName":"iisexpress","ThreadId":293,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"ERROR"}
{"@t":"2022-02-22T08:28:51.0839008Z","@mt":"Failed (will repeat).","@l":"Error","@x":"System.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server) ---> System.ComponentModel.Win32Exception (0x80004005): The network path was not found\r\n   at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)\r\n   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)\r\n   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)\r\n   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)\r\n   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.Open()\r\n   at NPoco.Database.OpenSharedConnectionImp(Boolean isInternal)\r\n   at NPoco.Database.BeginTransaction(IsolationLevel isolationLevel)\r\n   at Umbraco.Core.Scoping.Scope.get_Database() in D:\\a\\1\\s\\src\\Umbraco.Core\\Scoping\\Scope.cs:line 330\r\n   at Umbraco.Core.Sync.DatabaseServerMessenger.Sync() in D:\\a\\1\\s\\src\\Umbraco.Core\\Sync\\DatabaseServerMessenger.cs:line 263\r\n   at Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent.InstructionProcessTask.PerformRun() in D:\\a\\1\\s\\src\\Umbraco.Web\\Compose\\DatabaseServerRegistrarAndMessengerComponent.cs:line 163\r\nClientConnectionId:00000000-0000-0000-0000-000000000000\r\nError Number:53,State:0,Class:20","SourceContext":"Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent+InstructionProcessTask","ProcessId":24244,"ProcessName":"iisexpress","ThreadId":321,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"ERROR"}
{"@t":"2022-02-22T08:28:56.0852426Z","@mt":"Failed (will repeat).","@l":"Error","@x":"System.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server) ---> System.ComponentModel.Win32Exception (0x80004005): The network path was not found\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)\r\n   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.Open()\r\n   at NPoco.Database.OpenSharedConnectionImp(Boolean isInternal)\r\n   at NPoco.Database.BeginTransaction(IsolationLevel isolationLevel)\r\n   at Umbraco.Core.Scoping.Scope.get_Database() in D:\\a\\1\\s\\src\\Umbraco.Core\\Scoping\\Scope.cs:line 330\r\n   at Umbraco.Core.Sync.DatabaseServerMessenger.Sync() in D:\\a\\1\\s\\src\\Umbraco.Core\\Sync\\DatabaseServerMessenger.cs:line 263\r\n   at Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent.InstructionProcessTask.PerformRun() in D:\\a\\1\\s\\src\\Umbraco.Web\\Compose\\DatabaseServerRegistrarAndMessengerComponent.cs:line 163\r\nClientConnectionId:00000000-0000-0000-0000-000000000000\r\nError Number:53,State:0,Class:20","SourceContext":"Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent+InstructionProcessTask","ProcessId":24244,"ProcessName":"iisexpress","ThreadId":283,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"ERROR"}
{"@t":"2022-02-22T08:29:01.0965915Z","@mt":"Failed (will repeat).","@l":"Error","@x":"System.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server) ---> System.ComponentModel.Win32Exception (0x80004005): The network path was not found\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)\r\n   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.Open()\r\n   at NPoco.Database.OpenSharedConnectionImp(Boolean isInternal)\r\n   at NPoco.Database.BeginTransaction(IsolationLevel isolationLevel)\r\n   at Umbraco.Core.Scoping.Scope.get_Database() in D:\\a\\1\\s\\src\\Umbraco.Core\\Scoping\\Scope.cs:line 330\r\n   at Umbraco.Core.Sync.DatabaseServerMessenger.Sync() in D:\\a\\1\\s\\src\\Umbraco.Core\\Sync\\DatabaseServerMessenger.cs:line 263\r\n   at Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent.InstructionProcessTask.PerformRun() in D:\\a\\1\\s\\src\\Umbraco.Web\\Compose\\DatabaseServerRegistrarAndMessengerComponent.cs:line 163\r\nClientConnectionId:00000000-0000-0000-0000-000000000000\r\nError Number:53,State:0,Class:20","SourceContext":"Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent+InstructionProcessTask","ProcessId":24244,"ProcessName":"iisexpress","ThreadId":283,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"ERROR"}
{"@t":"2022-02-22T08:29:06.1122287Z","@mt":"Failed (will repeat).","@l":"Error","@x":"System.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server) ---> System.ComponentModel.Win32Exception (0x80004005): The network path was not found\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)\r\n   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.Open()\r\n   at NPoco.Database.OpenSharedConnectionImp(Boolean isInternal)\r\n   at NPoco.Database.BeginTransaction(IsolationLevel isolationLevel)\r\n   at Umbraco.Core.Scoping.Scope.get_Database() in D:\\a\\1\\s\\src\\Umbraco.Core\\Scoping\\Scope.cs:line 330\r\n   at Umbraco.Core.Sync.DatabaseServerMessenger.Sync() in D:\\a\\1\\s\\src\\Umbraco.Core\\Sync\\DatabaseServerMessenger.cs:line 263\r\n   at Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent.InstructionProcessTask.PerformRun() in D:\\a\\1\\s\\src\\Umbraco.Web\\Compose\\DatabaseServerRegistrarAndMessengerComponent.cs:line 163\r\nClientConnectionId:00000000-0000-0000-0000-000000000000\r\nError Number:53,State:0,Class:20","SourceContext":"Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent+InstructionProcessTask","ProcessId":24244,"ProcessName":"iisexpress","ThreadId":320,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"ERROR"}
{"@t":"2022-02-22T08:29:25.7480694Z","@mt":"Failed (will repeat).","@l":"Error","@x":"System.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server) ---> System.ComponentModel.Win32Exception (0x80004005): The network path was not found\r\n   at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)\r\n   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)\r\n   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)\r\n   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)\r\n   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.Open()\r\n   at NPoco.Database.OpenSharedConnectionImp(Boolean isInternal)\r\n   at NPoco.Database.BeginTransaction(IsolationLevel isolationLevel)\r\n   at Umbraco.Core.Scoping.Scope.get_Database() in D:\\a\\1\\s\\src\\Umbraco.Core\\Scoping\\Scope.cs:line 330\r\n   at Umbraco.Core.Sync.DatabaseServerMessenger.Sync() in D:\\a\\1\\s\\src\\Umbraco.Core\\Sync\\DatabaseServerMessenger.cs:line 263\r\n   at Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent.InstructionProcessTask.PerformRun() in D:\\a\\1\\s\\src\\Umbraco.Web\\Compose\\DatabaseServerRegistrarAndMessengerComponent.cs:line 163\r\nClientConnectionId:00000000-0000-0000-0000-000000000000\r\nError Number:53,State:0,Class:20","SourceContext":"Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent+InstructionProcessTask","ProcessId":24244,"ProcessName":"iisexpress","ThreadId":293,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"ERROR"}
{"@t":"2022-02-22T08:29:30.7490696Z","@mt":"Failed (will repeat).","@l":"Error","@x":"System.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server) ---> System.ComponentModel.Win32Exception (0x80004005): The network path was not found\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)\r\n   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.Open()\r\n   at NPoco.Database.OpenSharedConnectionImp(Boolean isInternal)\r\n   at NPoco.Database.BeginTransaction(IsolationLevel isolationLevel)\r\n   at Umbraco.Core.Scoping.Scope.get_Database() in D:\\a\\1\\s\\src\\Umbraco.Core\\Scoping\\Scope.cs:line 330\r\n   at Umbraco.Core.Sync.DatabaseServerMessenger.Sync() in D:\\a\\1\\s\\src\\Umbraco.Core\\Sync\\DatabaseServerMessenger.cs:line 263\r\n   at Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent.InstructionProcessTask.PerformRun() in D:\\a\\1\\s\\src\\Umbraco.Web\\Compose\\DatabaseServerRegistrarAndMessengerComponent.cs:line 163\r\nClientConnectionId:00000000-0000-0000-0000-000000000000\r\nError Number:53,State:0,Class:20","SourceContext":"Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent+InstructionProcessTask","ProcessId":24244,"ProcessName":"iisexpress","ThreadId":289,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"ERROR"}
{"@t":"2022-02-22T08:29:35.7502774Z","@mt":"Failed (will repeat).","@l":"Error","@x":"System.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server) ---> System.ComponentModel.Win32Exception (0x80004005): The network path was not found\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)\r\n   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.Open()\r\n   at NPoco.Database.OpenSharedConnectionImp(Boolean isInternal)\r\n   at NPoco.Database.BeginTransaction(IsolationLevel isolationLevel)\r\n   at Umbraco.Core.Scoping.Scope.get_Database() in D:\\a\\1\\s\\src\\Umbraco.Core\\Scoping\\Scope.cs:line 330\r\n   at Umbraco.Core.Sync.DatabaseServerMessenger.Sync() in D:\\a\\1\\s\\src\\Umbraco.Core\\Sync\\DatabaseServerMessenger.cs:line 263\r\n   at Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent.InstructionProcessTask.PerformRun() in D:\\a\\1\\s\\src\\Umbraco.Web\\Compose\\DatabaseServerRegistrarAndMessengerComponent.cs:line 163\r\nClientConnectionId:00000000-0000-0000-0000-000000000000\r\nError Number:53,State:0,Class:20","SourceContext":"Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent+InstructionProcessTask","ProcessId":24244,"ProcessName":"iisexpress","ThreadId":288,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"ERROR"}
{"@t":"2022-02-22T08:29:40.7652822Z","@mt":"Failed (will repeat).","@l":"Error","@x":"System.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server) ---> System.ComponentModel.Win32Exception (0x80004005): The network path was not found\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)\r\n   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.Open()\r\n   at NPoco.Database.OpenSharedConnectionImp(Boolean isInternal)\r\n   at NPoco.Database.BeginTransaction(IsolationLevel isolationLevel)\r\n   at Umbraco.Core.Scoping.Scope.get_Database() in D:\\a\\1\\s\\src\\Umbraco.Core\\Scoping\\Scope.cs:line 330\r\n   at Umbraco.Core.Sync.DatabaseServerMessenger.Sync() in D:\\a\\1\\s\\src\\Umbraco.Core\\Sync\\DatabaseServerMessenger.cs:line 263\r\n   at Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent.InstructionProcessTask.PerformRun() in D:\\a\\1\\s\\src\\Umbraco.Web\\Compose\\DatabaseServerRegistrarAndMessengerComponent.cs:line 163\r\nClientConnectionId:00000000-0000-0000-0000-000000000000\r\nError Number:53,State:0,Class:20","SourceContext":"Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent+InstructionProcessTask","ProcessId":24244,"ProcessName":"iisexpress","ThreadId":283,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"ERROR"}
{"@t":"2022-02-22T08:29:50.7770914Z","@mt":"Failed (will repeat).","@l":"Error","@x":"System.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server) ---> System.ComponentModel.Win32Exception (0x80004005): The network path was not found\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)\r\n   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.Open()\r\n   at NPoco.Database.OpenSharedConnectionImp(Boolean isInternal)\r\n   at NPoco.Database.BeginTransaction(IsolationLevel isolationLevel)\r\n   at Umbraco.Core.Scoping.Scope.get_Database() in D:\\a\\1\\s\\src\\Umbraco.Core\\Scoping\\Scope.cs:line 330\r\n   at Umbraco.Core.Sync.DatabaseServerMessenger.Sync() in D:\\a\\1\\s\\src\\Umbraco.Core\\Sync\\DatabaseServerMessenger.cs:line 263\r\n   at Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent.InstructionProcessTask.PerformRun() in D:\\a\\1\\s\\src\\Umbraco.Web\\Compose\\DatabaseServerRegistrarAndMessengerComponent.cs:line 163\r\nClientConnectionId:00000000-0000-0000-0000-000000000000\r\nError Number:53,State:0,Class:20","SourceContext":"Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent+InstructionProcessTask","ProcessId":24244,"ProcessName":"iisexpress","ThreadId":299,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"ERROR"}
{"@t":"2022-02-22T08:29:55.7901321Z","@mt":"Failed (will repeat).","@l":"Error","@x":"System.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server) ---> System.ComponentModel.Win32Exception (0x80004005): The network path was not found\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)\r\n   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.Open()\r\n   at NPoco.Database.OpenSharedConnectionImp(Boolean isInternal)\r\n   at NPoco.Database.BeginTransaction(IsolationLevel isolationLevel)\r\n   at Umbraco.Core.Scoping.Scope.get_Database() in D:\\a\\1\\s\\src\\Umbraco.Core\\Scoping\\Scope.cs:line 330\r\n   at Umbraco.Core.Sync.DatabaseServerMessenger.Sync() in D:\\a\\1\\s\\src\\Umbraco.Core\\Sync\\DatabaseServerMessenger.cs:line 263\r\n   at Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent.InstructionProcessTask.PerformRun() in D:\\a\\1\\s\\src\\Umbraco.Web\\Compose\\DatabaseServerRegistrarAndMessengerComponent.cs:line 163\r\nClientConnectionId:00000000-0000-0000-0000-000000000000\r\nError Number:53,State:0,Class:20","SourceContext":"Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent+InstructionProcessTask","ProcessId":24244,"ProcessName":"iisexpress","ThreadId":315,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"ERROR"}
{"@t":"2022-02-22T08:30:00.7975751Z","@mt":"Failed (will repeat).","@l":"Error","@x":"System.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server) ---> System.ComponentModel.Win32Exception (0x80004005): The network path was not found\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)\r\n   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.Open()\r\n   at NPoco.Database.OpenSharedConnectionImp(Boolean isInternal)\r\n   at NPoco.Database.BeginTransaction(IsolationLevel isolationLevel)\r\n   at Umbraco.Core.Scoping.Scope.get_Database() in D:\\a\\1\\s\\src\\Umbraco.Core\\Scoping\\Scope.cs:line 330\r\n   at Umbraco.Core.Sync.DatabaseServerMessenger.Sync() in D:\\a\\1\\s\\src\\Umbraco.Core\\Sync\\DatabaseServerMessenger.cs:line 263\r\n   at Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent.InstructionProcessTask.PerformRun() in D:\\a\\1\\s\\src\\Umbraco.Web\\Compose\\DatabaseServerRegistrarAndMessengerComponent.cs:line 163\r\nClientConnectionId:00000000-0000-0000-0000-000000000000\r\nError Number:53,State:0,Class:20","SourceContext":"Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent+InstructionProcessTask","ProcessId":24244,"ProcessName":"iisexpress","ThreadId":321,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"ERROR"}
{"@t":"2022-02-22T08:39:02.0184462Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Booting Umbraco 8.17.2.","TimingId":"9ce8a43","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":21536,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"520ea2d9-263c-42d2-8033-4db7a25ef9e8"}
{"@t":"2022-02-22T08:39:02.0944454Z","@mt":"Booting site '{HostingSiteName}', app '{HostingApplicationID}', path '{HostingPhysicalPath}', server '{MachineName}'.","HostingSiteName":"Flight2Orbit","HostingApplicationID":"/LM/W3SVC/2/ROOT","HostingPhysicalPath":"D:\\Users\\Oliver.Gjesing.EMEA\\Documents\\Git\\FTO\\BE\\Flight2Orbit\\","MachineName":"AAR00118","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":21536,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"520ea2d9-263c-42d2-8033-4db7a25ef9e8"}
{"@t":"2022-02-22T08:39:02.5074762Z","@mt":"Acquiring.","SourceContext":"Umbraco.Core.Runtime.MainDom","ProcessId":21536,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"520ea2d9-263c-42d2-8033-4db7a25ef9e8"}
{"@t":"2022-02-22T08:39:02.5114465Z","@mt":"Acquired.","SourceContext":"Umbraco.Core.Runtime.MainDom","ProcessId":21536,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"520ea2d9-263c-42d2-8033-4db7a25ef9e8"}
{"@t":"2022-02-22T08:39:06.7754480Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Resolving composer types.","TimingId":"e3bafac","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":21536,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"520ea2d9-263c-42d2-8033-4db7a25ef9e8"}
{"@t":"2022-02-22T08:39:08.0474486Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Resolved.","Duration":1271,"TimingId":"e3bafac","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":21536,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"520ea2d9-263c-42d2-8033-4db7a25ef9e8"}
{"@t":"2022-02-22T08:39:32.9059645Z","@mt":"Profiler is VoidProfiler, not profiling (must run debug mode to profile).","SourceContext":"Umbraco.Web.Logging.WebProfilerComponent","ProcessId":21536,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"520ea2d9-263c-42d2-8033-4db7a25ef9e8"}
{"@t":"2022-02-22T08:39:33.2979809Z","@mt":"Initializing Umbraco internal event handlers for cache refreshing.","SourceContext":"Umbraco.Web.Cache.DistributedCacheBinderComponent","ProcessId":21536,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"520ea2d9-263c-42d2-8033-4db7a25ef9e8"}
{"@t":"2022-02-22T08:39:33.5769701Z","@mt":"Adding examine event handlers for {RegisteredIndexers} index providers.","RegisteredIndexers":3,"SourceContext":"Umbraco.Web.Search.ExamineComponent","ProcessId":21536,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"520ea2d9-263c-42d2-8033-4db7a25ef9e8"}
{"@t":"2022-02-22T08:39:33.8389697Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Booted.","Duration":31849,"TimingId":"9ce8a43","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":21536,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"520ea2d9-263c-42d2-8033-4db7a25ef9e8"}
{"@t":"2022-02-22T08:39:35.2929736Z","@mt":"New url {Url} detected, re-discovering application url.","Url":"http://localhost:54252/umbraco","SourceContext":"Umbraco.Core.Sync.ApplicationUrlHelper","ProcessId":21536,"ProcessName":"iisexpress","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"fc4c1c1d-51dd-43ae-8eed-ef95e56117e3"}
{"@t":"2022-02-22T08:39:35.2999711Z","@mt":"ApplicationUrl: {UmbracoAppUrl} (UmbracoModule request)","UmbracoAppUrl":"http://localhost:54252/umbraco","SourceContext":"Umbraco.Core.Sync.ApplicationUrlHelper","ProcessId":21536,"ProcessName":"iisexpress","ThreadId":6,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":3,"HttpRequestId":"9656b00d-2915-40f2-832f-5c7a9ab164dc"}
{"@t":"2022-02-22T08:39:35.2999711Z","@mt":"ApplicationUrl: {UmbracoAppUrl} (UmbracoModule request)","UmbracoAppUrl":"http://localhost:54252/umbraco","SourceContext":"Umbraco.Core.Sync.ApplicationUrlHelper","ProcessId":21536,"ProcessName":"iisexpress","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"fc4c1c1d-51dd-43ae-8eed-ef95e56117e3"}
{"@t":"2022-02-22T08:39:35.2999711Z","@mt":"ApplicationUrl: {UmbracoAppUrl} (UmbracoModule request)","UmbracoAppUrl":"http://localhost:54252/umbraco","SourceContext":"Umbraco.Core.Sync.ApplicationUrlHelper","ProcessId":21536,"ProcessName":"iisexpress","ThreadId":7,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":4,"HttpRequestId":"847ab888-72e4-4ad7-b22a-dcc566585426"}
{"@t":"2022-02-22T08:39:38.4839711Z","@mt":"Starting initialize async background thread.","SourceContext":"Umbraco.Web.Search.BackgroundIndexRebuilder","ProcessId":21536,"ProcessName":"iisexpress","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"fc4c1c1d-51dd-43ae-8eed-ef95e56117e3"}
{"@t":"2022-02-22T08:39:38.6239720Z","@mt":"Registered with MainDom, localContentDbExists? {LocalContentDbExists}, localMediaDbExists? {LocalMediaDbExists}","LocalContentDbExists":true,"LocalMediaDbExists":true,"SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":21536,"ProcessName":"iisexpress","ThreadId":7,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":4,"HttpRequestId":"847ab888-72e4-4ad7-b22a-dcc566585426"}
{"@t":"2022-02-22T08:39:38.6249772Z","@mt":"Creating the content store, localContentDbExists? {LocalContentDbExists}","LocalContentDbExists":true,"SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":21536,"ProcessName":"iisexpress","ThreadId":7,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":4,"HttpRequestId":"847ab888-72e4-4ad7-b22a-dcc566585426"}
{"@t":"2022-02-22T08:39:38.6459836Z","@mt":"Creating the media store, localMediaDbExists? {LocalMediaDbExists}","LocalMediaDbExists":true,"SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":21536,"ProcessName":"iisexpress","ThreadId":7,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":4,"HttpRequestId":"847ab888-72e4-4ad7-b22a-dcc566585426"}
{"@t":"2022-02-22T08:39:42.1899884Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Loading content from local cache file","TimingId":"6e60c8b","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":21536,"ProcessName":"iisexpress","ThreadId":7,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":4,"HttpRequestId":"847ab888-72e4-4ad7-b22a-dcc566585426"}
{"@t":"2022-02-22T08:39:42.3669781Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Completed.","Duration":176,"TimingId":"6e60c8b","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":21536,"ProcessName":"iisexpress","ThreadId":7,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":4,"HttpRequestId":"847ab888-72e4-4ad7-b22a-dcc566585426"}
{"@t":"2022-02-22T08:39:42.4419908Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Loading media from local cache file","TimingId":"214107d","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":21536,"ProcessName":"iisexpress","ThreadId":7,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":4,"HttpRequestId":"847ab888-72e4-4ad7-b22a-dcc566585426"}
{"@t":"2022-02-22T08:39:42.4479749Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Completed.","Duration":6,"TimingId":"214107d","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":21536,"ProcessName":"iisexpress","ThreadId":7,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":4,"HttpRequestId":"847ab888-72e4-4ad7-b22a-dcc566585426"}
{"@t":"2022-02-22T10:53:17.5242031Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Booting Umbraco 8.17.2.","TimingId":"f7d2a98","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"e0917e28-cebc-4e2f-b517-dadce494f765"}
{"@t":"2022-02-22T10:53:17.5641976Z","@mt":"Booting site '{HostingSiteName}', app '{HostingApplicationID}', path '{HostingPhysicalPath}', server '{MachineName}'.","HostingSiteName":"Flight2Orbit","HostingApplicationID":"/LM/W3SVC/2/ROOT","HostingPhysicalPath":"D:\\Users\\Oliver.Gjesing.EMEA\\Documents\\Git\\FTO\\BE\\Flight2Orbit\\","MachineName":"AAR00118","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"e0917e28-cebc-4e2f-b517-dadce494f765"}
{"@t":"2022-02-22T10:53:17.7574050Z","@mt":"Acquiring.","SourceContext":"Umbraco.Core.Runtime.MainDom","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"e0917e28-cebc-4e2f-b517-dadce494f765"}
{"@t":"2022-02-22T10:53:17.7614279Z","@mt":"Acquired.","SourceContext":"Umbraco.Core.Runtime.MainDom","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"e0917e28-cebc-4e2f-b517-dadce494f765"}
{"@t":"2022-02-22T10:53:19.9753913Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Resolving composer types.","TimingId":"ac84c62","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"e0917e28-cebc-4e2f-b517-dadce494f765"}
{"@t":"2022-02-22T10:53:20.6512567Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Resolved.","Duration":674,"TimingId":"ac84c62","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"e0917e28-cebc-4e2f-b517-dadce494f765"}
{"@t":"2022-02-22T10:53:30.2610150Z","@mt":"Profiler is VoidProfiler, not profiling (must run debug mode to profile).","SourceContext":"Umbraco.Web.Logging.WebProfilerComponent","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"e0917e28-cebc-4e2f-b517-dadce494f765"}
{"@t":"2022-02-22T10:53:30.4112948Z","@mt":"Initializing Umbraco internal event handlers for cache refreshing.","SourceContext":"Umbraco.Web.Cache.DistributedCacheBinderComponent","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"e0917e28-cebc-4e2f-b517-dadce494f765"}
{"@t":"2022-02-22T10:53:30.5114037Z","@mt":"Adding examine event handlers for {RegisteredIndexers} index providers.","RegisteredIndexers":3,"SourceContext":"Umbraco.Web.Search.ExamineComponent","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"e0917e28-cebc-4e2f-b517-dadce494f765"}
{"@t":"2022-02-22T10:53:30.5772054Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Booted.","Duration":13054,"TimingId":"f7d2a98","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"e0917e28-cebc-4e2f-b517-dadce494f765"}
{"@t":"2022-02-22T10:53:31.0429337Z","@mt":"New url {Url} detected, re-discovering application url.","Url":"http://localhost:54252/umbraco","SourceContext":"Umbraco.Core.Sync.ApplicationUrlHelper","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":12,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"65dbda90-3ee8-42fc-bd93-5884312c2eb4"}
{"@t":"2022-02-22T10:53:31.0449337Z","@mt":"ApplicationUrl: {UmbracoAppUrl} (UmbracoModule request)","UmbracoAppUrl":"http://localhost:54252/umbraco","SourceContext":"Umbraco.Core.Sync.ApplicationUrlHelper","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":12,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"65dbda90-3ee8-42fc-bd93-5884312c2eb4"}
{"@t":"2022-02-22T10:53:31.7088459Z","@mt":"Registered with MainDom, localContentDbExists? {LocalContentDbExists}, localMediaDbExists? {LocalMediaDbExists}","LocalContentDbExists":true,"LocalMediaDbExists":true,"SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":12,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"65dbda90-3ee8-42fc-bd93-5884312c2eb4"}
{"@t":"2022-02-22T10:53:31.7088459Z","@mt":"Creating the content store, localContentDbExists? {LocalContentDbExists}","LocalContentDbExists":true,"SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":12,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"65dbda90-3ee8-42fc-bd93-5884312c2eb4"}
{"@t":"2022-02-22T10:53:31.7108485Z","@mt":"Creating the media store, localMediaDbExists? {LocalMediaDbExists}","LocalMediaDbExists":true,"SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":12,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"65dbda90-3ee8-42fc-bd93-5884312c2eb4"}
{"@t":"2022-02-22T10:53:32.5027582Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Loading content from local cache file","TimingId":"8952cac","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":12,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"65dbda90-3ee8-42fc-bd93-5884312c2eb4"}
{"@t":"2022-02-22T10:53:32.5377940Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Completed.","Duration":35,"TimingId":"8952cac","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":12,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"65dbda90-3ee8-42fc-bd93-5884312c2eb4"}
{"@t":"2022-02-22T10:53:32.5518103Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Loading media from local cache file","TimingId":"4917263","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":12,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"65dbda90-3ee8-42fc-bd93-5884312c2eb4"}
{"@t":"2022-02-22T10:53:32.5538130Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Completed.","Duration":1,"TimingId":"4917263","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":12,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"65dbda90-3ee8-42fc-bd93-5884312c2eb4"}
{"@t":"2022-02-22T10:53:32.8263388Z","@mt":"Forcing index {IndexerName} to be unlocked since it was left in a locked state","IndexerName":"InternalIndex","SourceContext":"Umbraco.Examine.ExamineExtensions","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":12,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"65dbda90-3ee8-42fc-bd93-5884312c2eb4"}
{"@t":"2022-02-22T10:53:32.8273406Z","@mt":"Forcing index {IndexerName} to be unlocked since it was left in a locked state","IndexerName":"MembersIndex","SourceContext":"Umbraco.Examine.ExamineExtensions","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":12,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"65dbda90-3ee8-42fc-bd93-5884312c2eb4"}
{"@t":"2022-02-22T10:53:32.8283414Z","@mt":"Forcing index {IndexerName} to be unlocked since it was left in a locked state","IndexerName":"ExternalIndex","SourceContext":"Umbraco.Examine.ExamineExtensions","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":12,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"65dbda90-3ee8-42fc-bd93-5884312c2eb4"}
{"@t":"2022-02-22T10:53:32.8303458Z","@mt":"Starting initialize async background thread.","SourceContext":"Umbraco.Web.Search.BackgroundIndexRebuilder","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":12,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"65dbda90-3ee8-42fc-bd93-5884312c2eb4"}
{"@t":"2022-02-22T11:33:46.2371286Z","@mt":"[{EventType}] Event Id: {EventId}, State: {State}","EventType":"Information","EventId":0,"State":"Login attempt succeeded for username oliver.gjesing@akqa.com from IP address ::1","SourceContext":"Umbraco.Core.Logging.OwinLogger","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":97,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":3,"HttpRequestId":"a8ad6bc9-cb69-4567-8d4c-6bf716d3b7c3"}
{"@t":"2022-02-22T11:33:46.2381234Z","@mt":"[{EventType}] Event Id: {EventId}, State: {State}","EventType":"Information","EventId":0,"State":"User: oliver.gjesing@akqa.com logged in from IP address ::1","SourceContext":"Umbraco.Core.Logging.OwinLogger","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":97,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":3,"HttpRequestId":"a8ad6bc9-cb69-4567-8d4c-6bf716d3b7c3"}
{"@t":"2022-02-22T11:43:47.6153443Z","@mt":"Exception","@l":"Error","@x":"ImageProcessor.Common.Exceptions.ImageProcessingException: ProcessImageAsync 609 : No image exists at D:\\Users\\Oliver.Gjesing.EMEA\\Documents\\Git\\FTO\\BE\\Flight2Orbit\\media\\bddkxw1w\\9368855062_a548b9c5eb_o.jpg","SourceContext":"ImageProcessor.Web.HttpModules.ImageProcessingModule","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":153,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"ERROR","HttpRequestNumber":4,"HttpRequestId":"6fab1296-d8e4-4e89-b893-14643a412c07"}
{"@t":"2022-02-22T11:59:16.5090047Z","@mt":"{LogPrefix} Terminating {Immediate}","LogPrefix":"[LogScrubber] ","Immediate":"","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":95,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO "}
{"@t":"2022-02-22T11:59:16.5100049Z","@mt":"{LogPrefix} Terminating {Immediate}","LogPrefix":"[TempFileCleanup] ","Immediate":"","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":95,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO "}
{"@t":"2022-02-22T11:59:16.5100049Z","@mt":"{LogPrefix} Terminating {Immediate}","LogPrefix":"[TelemetryReporter] ","Immediate":"","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":95,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO "}
{"@t":"2022-02-22T11:59:16.5190056Z","@mt":"Stopping ({SignalSource})","SignalSource":"environment","SourceContext":"Umbraco.Core.Runtime.MainDom","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":95,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO "}
{"@t":"2022-02-22T11:59:16.5190056Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[TelemetryReporter] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":159,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO "}
{"@t":"2022-02-22T11:59:16.5200040Z","@mt":"{LogPrefix} Tasks {TaskStatus}, terminated","LogPrefix":"[TelemetryReporter] ","TaskStatus":"cancelled","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":159,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO "}
{"@t":"2022-02-22T11:59:16.5220046Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[LogScrubber] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":121,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO "}
{"@t":"2022-02-22T11:59:16.5220046Z","@mt":"{LogPrefix} Tasks {TaskStatus}, terminated","LogPrefix":"[LogScrubber] ","TaskStatus":"cancelled","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":121,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO "}
{"@t":"2022-02-22T11:59:16.5230076Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[TempFileCleanup] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":116,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO "}
{"@t":"2022-02-22T11:59:16.5230076Z","@mt":"{LogPrefix} Tasks {TaskStatus}, terminated","LogPrefix":"[TempFileCleanup] ","TaskStatus":"cancelled","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":116,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO "}
{"@t":"2022-02-22T11:59:16.5370048Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Examine shutting down","TimingId":"2b86fc4","SourceContext":"Umbraco.Web.Search.ExamineComponent","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":95,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO "}
{"@t":"2022-02-22T11:59:16.5470089Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Completed.","Duration":9,"TimingId":"2b86fc4","SourceContext":"Umbraco.Web.Search.ExamineComponent","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":95,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO "}
{"@t":"2022-02-22T11:59:16.5960088Z","@mt":"Released from MainDom","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":95,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO "}
{"@t":"2022-02-22T11:59:16.5980011Z","@mt":"Released ({SignalSource})","SignalSource":"environment","SourceContext":"Umbraco.Core.Runtime.MainDom","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":95,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO "}
{"@t":"2022-02-22T11:59:16.5980011Z","@mt":"{LogPrefix} Terminating {Immediate}","LogPrefix":"[ServerInstProcess] ","Immediate":"","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":95,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO "}
{"@t":"2022-02-22T11:59:16.5980011Z","@mt":"{LogPrefix} Terminating {Immediate}","LogPrefix":"[RebuildIndexesOnStartup] ","Immediate":"","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":95,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO "}
{"@t":"2022-02-22T11:59:16.5980011Z","@mt":"{LogPrefix} Terminating {Immediate}","LogPrefix":"[ServerRegistration] ","Immediate":"","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":95,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO "}
{"@t":"2022-02-22T11:59:16.5980011Z","@mt":"{LogPrefix} Terminating {Immediate}","LogPrefix":"[HealthCheckNotifier] ","Immediate":"","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":95,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO "}
{"@t":"2022-02-22T11:59:16.5980011Z","@mt":"{LogPrefix} Terminating {Immediate}","LogPrefix":"[Umbraco.Web.Scheduling.IBackgroundTask] ","Immediate":"","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":95,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO "}
{"@t":"2022-02-22T11:59:16.5980011Z","@mt":"{LogPrefix} Terminating {Immediate}","LogPrefix":"[ScheduledPublishing] ","Immediate":"","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":95,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO "}
{"@t":"2022-02-22T11:59:16.5980011Z","@mt":"{LogPrefix} Terminating {Immediate}","LogPrefix":"[ScheduledTasks] ","Immediate":"","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":95,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO "}
{"@t":"2022-02-22T11:59:16.5980011Z","@mt":"{LogPrefix} Terminating {Immediate}","LogPrefix":"[KeepAlive] ","Immediate":"","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":95,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO "}
{"@t":"2022-02-22T11:59:16.5980011Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[RebuildIndexesOnStartup] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":159,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO "}
{"@t":"2022-02-22T11:59:16.5980011Z","@mt":"{LogPrefix} Tasks {TaskStatus}, terminated","LogPrefix":"[RebuildIndexesOnStartup] ","TaskStatus":"cancelled","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":159,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO "}
{"@t":"2022-02-22T11:59:17.4911249Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[Umbraco.Web.Scheduling.IBackgroundTask] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":164,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO "}
{"@t":"2022-02-22T11:59:17.4911249Z","@mt":"{LogPrefix} Tasks {TaskStatus}, terminated","LogPrefix":"[Umbraco.Web.Scheduling.IBackgroundTask] ","TaskStatus":"cancelled","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":164,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO "}
{"@t":"2022-02-22T11:59:18.4885699Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[ServerInstProcess] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":84,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO "}
{"@t":"2022-02-22T11:59:18.4885699Z","@mt":"{LogPrefix} Tasks {TaskStatus}, terminated","LogPrefix":"[ServerInstProcess] ","TaskStatus":"cancelled","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":84,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO "}
{"@t":"2022-02-22T11:59:18.5190855Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[LogScrubber] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":157,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO "}
{"@t":"2022-02-22T11:59:18.5190855Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[TempFileCleanup] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":165,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO "}
{"@t":"2022-02-22T11:59:18.5190855Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[TelemetryReporter] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":111,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO "}
{"@t":"2022-02-22T11:59:18.5190855Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[ScheduledTasks] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":111,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO "}
{"@t":"2022-02-22T11:59:18.5190855Z","@mt":"{LogPrefix} Tasks {TaskStatus}, terminated","LogPrefix":"[ScheduledTasks] ","TaskStatus":"cancelled","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":111,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO "}
{"@t":"2022-02-22T11:59:18.5190855Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[ServerRegistration] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":165,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO "}
{"@t":"2022-02-22T11:59:18.5190855Z","@mt":"{LogPrefix} Tasks {TaskStatus}, terminated","LogPrefix":"[ServerRegistration] ","TaskStatus":"cancelled","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":165,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO "}
{"@t":"2022-02-22T11:59:18.5190855Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[HealthCheckNotifier] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":165,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO "}
{"@t":"2022-02-22T11:59:18.5190855Z","@mt":"{LogPrefix} Tasks {TaskStatus}, terminated","LogPrefix":"[HealthCheckNotifier] ","TaskStatus":"cancelled","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":165,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO "}
{"@t":"2022-02-22T11:59:18.5190855Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[ScheduledPublishing] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":165,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO "}
{"@t":"2022-02-22T11:59:18.5190855Z","@mt":"{LogPrefix} Tasks {TaskStatus}, terminated","LogPrefix":"[ScheduledPublishing] ","TaskStatus":"cancelled","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":165,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO "}
{"@t":"2022-02-22T11:59:18.5190855Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[KeepAlive] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":165,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO "}
{"@t":"2022-02-22T11:59:18.5190855Z","@mt":"{LogPrefix} Tasks {TaskStatus}, terminated","LogPrefix":"[KeepAlive] ","TaskStatus":"cancelled","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":165,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO "}
{"@t":"2022-02-22T11:59:18.6132518Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[RebuildIndexesOnStartup] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":159,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO "}
{"@t":"2022-02-22T11:59:18.6132518Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[ServerInstProcess] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":163,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO "}
{"@t":"2022-02-22T11:59:18.6132518Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[HealthCheckNotifier] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":121,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO "}
{"@t":"2022-02-22T11:59:18.6132518Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[ServerRegistration] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":116,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO "}
{"@t":"2022-02-22T11:59:19.5035700Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[Umbraco.Web.Scheduling.IBackgroundTask] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":164,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO "}
{"@t":"2022-02-22T11:59:20.4992498Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[ScheduledPublishing] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":84,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO "}
{"@t":"2022-02-22T11:59:20.5312144Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[ScheduledTasks] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":111,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO "}
{"@t":"2022-02-22T11:59:20.5312144Z","@mt":"{LogPrefix} Canceling tasks","LogPrefix":"[KeepAlive] ","SourceContext":"Umbraco.Web.Scheduling.BackgroundTaskRunner","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":157,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO "}
{"@t":"2022-02-22T11:59:46.6694946Z","@mt":"Application shutdown. Details: {ShutdownReason}\r\n\r\n_shutDownMessage={ShutdownMessage}\r\n\r\n_shutDownStack={ShutdownStack}","ShutdownReason":"ConfigurationChange","ShutdownMessage":"CONFIG change\r\nIIS configuration change\r\nHostingEnvironment initiated shutdown\r\nCONFIG change\r\nCONFIG change\r\nChange Notification for critical directories.\r\nbin dir change or directory rename\r\nChange Notification for critical directories.\r\nbin dir change or directory rename\r\nChange Notification for critical directories.\r\nbin dir change or directory rename\r\nChange Notification for critical directories.\r\nbin dir change or directory rename\r\nChange Notification for critical directories.\r\nbin dir change or directory rename\r\nChange Notification for critical directories.\r\nbin dir change or directory rename\r\nHostingEnvironment caused shutdown","ShutdownStack":"   at System.Environment.GetStackTrace(Exception e, Boolean needFileInfo)\r\n   at System.Environment.get_StackTrace()\r\n   at System.Web.Hosting.HostingEnvironment.InitiateShutdownInternal()\r\n   at System.Web.Hosting.HostingEnvironment.InitiateShutdownWithoutDemand()\r\n   at System.Web.Hosting.PipelineRuntime.StopProcessing()","SourceContext":"Umbraco.Web.UmbracoApplicationBase","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":161,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO "}
{"@t":"2022-02-22T12:00:10.7144128Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Booting Umbraco 8.17.2.","TimingId":"c6dd3f4","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":172,"AppDomainId":3,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"fcfb0351-b3b5-4b33-82a3-f7caf990c0b1"}
{"@t":"2022-02-22T12:00:10.7334117Z","@mt":"Booting site '{HostingSiteName}', app '{HostingApplicationID}', path '{HostingPhysicalPath}', server '{MachineName}'.","HostingSiteName":"Flight2Orbit","HostingApplicationID":"/LM/W3SVC/2/ROOT","HostingPhysicalPath":"D:\\Users\\Oliver.Gjesing.EMEA\\Documents\\Git\\FTO\\BE\\Flight2Orbit\\","MachineName":"AAR00118","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":172,"AppDomainId":3,"AppDomainAppId":"LMW3SVC2ROOT","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"fcfb0351-b3b5-4b33-82a3-f7caf990c0b1"}
{"@t":"2022-02-22T12:00:10.8234116Z","@mt":"Acquiring.","SourceContext":"Umbraco.Core.Runtime.MainDom","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":172,"AppDomainId":3,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"fcfb0351-b3b5-4b33-82a3-f7caf990c0b1"}
{"@t":"2022-02-22T12:00:10.8254087Z","@mt":"Acquired.","SourceContext":"Umbraco.Core.Runtime.MainDom","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":172,"AppDomainId":3,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"fcfb0351-b3b5-4b33-82a3-f7caf990c0b1"}
{"@t":"2022-02-22T12:00:11.3003903Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Resolving composer types.","TimingId":"2242619","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":172,"AppDomainId":3,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"fcfb0351-b3b5-4b33-82a3-f7caf990c0b1"}
{"@t":"2022-02-22T12:00:11.5334184Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Resolved.","Duration":231,"TimingId":"2242619","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":172,"AppDomainId":3,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"fcfb0351-b3b5-4b33-82a3-f7caf990c0b1"}
{"@t":"2022-02-22T12:00:12.8963322Z","@mt":"Initializing Umbraco internal event handlers for cache refreshing.","SourceContext":"Umbraco.Web.Cache.DistributedCacheBinderComponent","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":172,"AppDomainId":3,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"fcfb0351-b3b5-4b33-82a3-f7caf990c0b1"}
{"@t":"2022-02-22T12:00:13.1503207Z","@mt":"Adding examine event handlers for {RegisteredIndexers} index providers.","RegisteredIndexers":3,"SourceContext":"Umbraco.Web.Search.ExamineComponent","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":172,"AppDomainId":3,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"fcfb0351-b3b5-4b33-82a3-f7caf990c0b1"}
{"@t":"2022-02-22T12:00:13.3233136Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Booted.","Duration":2624,"TimingId":"c6dd3f4","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":172,"AppDomainId":3,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"fcfb0351-b3b5-4b33-82a3-f7caf990c0b1"}
{"@t":"2022-02-22T12:00:14.1862906Z","@mt":"New url {Url} detected, re-discovering application url.","Url":"http://localhost:54252/umbraco","SourceContext":"Umbraco.Core.Sync.ApplicationUrlHelper","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":174,"AppDomainId":3,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"ff0364e7-6b6f-4f30-b984-8221bd8cfc3d"}
{"@t":"2022-02-22T12:00:14.1922827Z","@mt":"ApplicationUrl: {UmbracoAppUrl} (UmbracoModule request)","UmbracoAppUrl":"http://localhost:54252/umbraco","SourceContext":"Umbraco.Core.Sync.ApplicationUrlHelper","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":174,"AppDomainId":3,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"ff0364e7-6b6f-4f30-b984-8221bd8cfc3d"}
{"@t":"2022-02-22T12:00:15.3782315Z","@mt":"Registered with MainDom, localContentDbExists? {LocalContentDbExists}, localMediaDbExists? {LocalMediaDbExists}","LocalContentDbExists":true,"LocalMediaDbExists":true,"SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":121,"AppDomainId":3,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":3,"HttpRequestId":"dc8bf602-6880-4208-bfc8-d2c12613c67c"}
{"@t":"2022-02-22T12:00:15.3792360Z","@mt":"Creating the content store, localContentDbExists? {LocalContentDbExists}","LocalContentDbExists":true,"SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":121,"AppDomainId":3,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":3,"HttpRequestId":"dc8bf602-6880-4208-bfc8-d2c12613c67c"}
{"@t":"2022-02-22T12:00:15.3822287Z","@mt":"Creating the media store, localMediaDbExists? {LocalMediaDbExists}","LocalMediaDbExists":true,"SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":121,"AppDomainId":3,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":3,"HttpRequestId":"dc8bf602-6880-4208-bfc8-d2c12613c67c"}
{"@t":"2022-02-22T12:00:15.4382264Z","@mt":"No last synced Id found, this generally means this is a new server/install. The server will build its caches and indexes, and then adjust its last synced Id to the latest found in the database and maintain cache updates based on that Id.","@l":"Warning","SourceContext":"Umbraco.Core.Sync.DatabaseServerMessenger","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":121,"AppDomainId":3,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"WARN ","HttpRequestNumber":3,"HttpRequestId":"dc8bf602-6880-4208-bfc8-d2c12613c67c"}
{"@t":"2022-02-22T12:00:17.2732484Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Loading content from database","TimingId":"bf27b3e","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":121,"AppDomainId":3,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":3,"HttpRequestId":"dc8bf602-6880-4208-bfc8-d2c12613c67c"}
{"@t":"2022-02-22T12:00:18.4892198Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Completed.","Duration":1216,"TimingId":"bf27b3e","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":121,"AppDomainId":3,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":3,"HttpRequestId":"dc8bf602-6880-4208-bfc8-d2c12613c67c"}
{"@t":"2022-02-22T12:00:18.6322097Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Loading media from database","TimingId":"32d00bf","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":121,"AppDomainId":3,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":3,"HttpRequestId":"dc8bf602-6880-4208-bfc8-d2c12613c67c"}
{"@t":"2022-02-22T12:00:18.7541960Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Completed.","Duration":121,"TimingId":"32d00bf","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":121,"AppDomainId":3,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":3,"HttpRequestId":"dc8bf602-6880-4208-bfc8-d2c12613c67c"}
{"@t":"2022-02-22T12:00:19.1441780Z","@mt":"Forcing index {IndexerName} to be unlocked since it was left in a locked state","IndexerName":"InternalIndex","SourceContext":"Umbraco.Examine.ExamineExtensions","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":161,"AppDomainId":3,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":4,"HttpRequestId":"8ef8b953-6f9a-4487-9c85-1831137e3f8d"}
{"@t":"2022-02-22T12:00:19.1471757Z","@mt":"Forcing index {IndexerName} to be unlocked since it was left in a locked state","IndexerName":"MembersIndex","SourceContext":"Umbraco.Examine.ExamineExtensions","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":161,"AppDomainId":3,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":4,"HttpRequestId":"8ef8b953-6f9a-4487-9c85-1831137e3f8d"}
{"@t":"2022-02-22T12:00:19.1481770Z","@mt":"Forcing index {IndexerName} to be unlocked since it was left in a locked state","IndexerName":"ExternalIndex","SourceContext":"Umbraco.Examine.ExamineExtensions","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":161,"AppDomainId":3,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":4,"HttpRequestId":"8ef8b953-6f9a-4487-9c85-1831137e3f8d"}
{"@t":"2022-02-22T12:00:19.1511770Z","@mt":"Starting initialize async background thread.","SourceContext":"Umbraco.Web.Search.BackgroundIndexRebuilder","ProcessId":1476,"ProcessName":"iisexpress","ThreadId":161,"AppDomainId":3,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":4,"HttpRequestId":"8ef8b953-6f9a-4487-9c85-1831137e3f8d"}
{"@t":"2022-02-22T16:34:25.8343704Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Booting Umbraco 8.17.2.","TimingId":"70da3db","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":13528,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"01312bab-2907-4236-a4f5-c1ddfd882633"}
{"@t":"2022-02-22T16:34:25.9141477Z","@mt":"Booting site '{HostingSiteName}', app '{HostingApplicationID}', path '{HostingPhysicalPath}', server '{MachineName}'.","HostingSiteName":"Flight2Orbit","HostingApplicationID":"/LM/W3SVC/2/ROOT","HostingPhysicalPath":"D:\\Users\\Oliver.Gjesing.EMEA\\Documents\\Git\\FTO\\BE\\Flight2Orbit\\","MachineName":"AAR00118","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":13528,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"01312bab-2907-4236-a4f5-c1ddfd882633"}
{"@t":"2022-02-22T16:34:26.2462799Z","@mt":"Acquiring.","SourceContext":"Umbraco.Core.Runtime.MainDom","ProcessId":13528,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"01312bab-2907-4236-a4f5-c1ddfd882633"}
{"@t":"2022-02-22T16:34:26.2541236Z","@mt":"Acquired.","SourceContext":"Umbraco.Core.Runtime.MainDom","ProcessId":13528,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"01312bab-2907-4236-a4f5-c1ddfd882633"}
{"@t":"2022-02-22T16:34:28.9980785Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Resolving composer types.","TimingId":"5fe564d","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":13528,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"01312bab-2907-4236-a4f5-c1ddfd882633"}
{"@t":"2022-02-22T16:34:48.0495092Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Resolved.","Duration":19055,"TimingId":"5fe564d","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":13528,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"01312bab-2907-4236-a4f5-c1ddfd882633"}
{"@t":"2022-02-22T16:34:50.3131692Z","@mt":"Initializing Umbraco internal event handlers for cache refreshing.","SourceContext":"Umbraco.Web.Cache.DistributedCacheBinderComponent","ProcessId":13528,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"01312bab-2907-4236-a4f5-c1ddfd882633"}
{"@t":"2022-02-22T16:34:50.5699195Z","@mt":"Adding examine event handlers for {RegisteredIndexers} index providers.","RegisteredIndexers":3,"SourceContext":"Umbraco.Web.Search.ExamineComponent","ProcessId":13528,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"01312bab-2907-4236-a4f5-c1ddfd882633"}
{"@t":"2022-02-22T16:34:50.7825568Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Booted.","Duration":24984,"TimingId":"70da3db","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":13528,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"01312bab-2907-4236-a4f5-c1ddfd882633"}
{"@t":"2022-02-22T16:34:52.0240673Z","@mt":"New url {Url} detected, re-discovering application url.","Url":"http://localhost:54252/umbraco","SourceContext":"Umbraco.Core.Sync.ApplicationUrlHelper","ProcessId":13528,"ProcessName":"iisexpress","ThreadId":3,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"b87d127d-4ef7-4e3a-acf3-d08898427910"}
{"@t":"2022-02-22T16:34:52.0240673Z","@mt":"New url {Url} detected, re-discovering application url.","Url":"http://localhost:54252/umbraco","SourceContext":"Umbraco.Core.Sync.ApplicationUrlHelper","ProcessId":13528,"ProcessName":"iisexpress","ThreadId":12,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":3,"HttpRequestId":"ae311b6d-87e4-408f-b939-7bffc4f7b85f"}
{"@t":"2022-02-22T16:34:52.0240673Z","@mt":"New url {Url} detected, re-discovering application url.","Url":"http://localhost:54252/umbraco","SourceContext":"Umbraco.Core.Sync.ApplicationUrlHelper","ProcessId":13528,"ProcessName":"iisexpress","ThreadId":13,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":4,"HttpRequestId":"d76728d0-93a7-4162-9082-906cef040d28"}
{"@t":"2022-02-22T16:34:52.0240673Z","@mt":"New url {Url} detected, re-discovering application url.","Url":"http://localhost:54252/umbraco","SourceContext":"Umbraco.Core.Sync.ApplicationUrlHelper","ProcessId":13528,"ProcessName":"iisexpress","ThreadId":14,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":5,"HttpRequestId":"c111e35f-af5f-4a6d-be01-b700b947c7a4"}
{"@t":"2022-02-22T16:34:52.0240673Z","@mt":"New url {Url} detected, re-discovering application url.","Url":"http://localhost:54252/umbraco","SourceContext":"Umbraco.Core.Sync.ApplicationUrlHelper","ProcessId":13528,"ProcessName":"iisexpress","ThreadId":5,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":6,"HttpRequestId":"83933c23-aa49-46f2-aa48-6bc67d61082c"}
{"@t":"2022-02-22T16:34:52.0240673Z","@mt":"New url {Url} detected, re-discovering application url.","Url":"http://localhost:54252/umbraco","SourceContext":"Umbraco.Core.Sync.ApplicationUrlHelper","ProcessId":13528,"ProcessName":"iisexpress","ThreadId":4,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":7,"HttpRequestId":"e675ec2c-06f8-41be-b688-af005b588487"}
{"@t":"2022-02-22T16:34:52.0240673Z","@mt":"New url {Url} detected, re-discovering application url.","Url":"http://localhost:54252/umbraco","SourceContext":"Umbraco.Core.Sync.ApplicationUrlHelper","ProcessId":13528,"ProcessName":"iisexpress","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":8,"HttpRequestId":"4b4871c9-b0fd-48ec-99ce-ed4ed543a6c3"}
{"@t":"2022-02-22T16:34:52.0260673Z","@mt":"ApplicationUrl: {UmbracoAppUrl} (UmbracoModule request)","UmbracoAppUrl":"http://localhost:54252/umbraco","SourceContext":"Umbraco.Core.Sync.ApplicationUrlHelper","ProcessId":13528,"ProcessName":"iisexpress","ThreadId":3,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"b87d127d-4ef7-4e3a-acf3-d08898427910"}
{"@t":"2022-02-22T16:34:52.0260673Z","@mt":"ApplicationUrl: {UmbracoAppUrl} (UmbracoModule request)","UmbracoAppUrl":"http://localhost:54252/umbraco","SourceContext":"Umbraco.Core.Sync.ApplicationUrlHelper","ProcessId":13528,"ProcessName":"iisexpress","ThreadId":12,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":3,"HttpRequestId":"ae311b6d-87e4-408f-b939-7bffc4f7b85f"}
{"@t":"2022-02-22T16:34:52.0260673Z","@mt":"ApplicationUrl: {UmbracoAppUrl} (UmbracoModule request)","UmbracoAppUrl":"http://localhost:54252/umbraco","SourceContext":"Umbraco.Core.Sync.ApplicationUrlHelper","ProcessId":13528,"ProcessName":"iisexpress","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":8,"HttpRequestId":"4b4871c9-b0fd-48ec-99ce-ed4ed543a6c3"}
{"@t":"2022-02-22T16:34:52.0260673Z","@mt":"ApplicationUrl: {UmbracoAppUrl} (UmbracoModule request)","UmbracoAppUrl":"http://localhost:54252/umbraco","SourceContext":"Umbraco.Core.Sync.ApplicationUrlHelper","ProcessId":13528,"ProcessName":"iisexpress","ThreadId":13,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":4,"HttpRequestId":"d76728d0-93a7-4162-9082-906cef040d28"}
{"@t":"2022-02-22T16:34:52.0260673Z","@mt":"ApplicationUrl: {UmbracoAppUrl} (UmbracoModule request)","UmbracoAppUrl":"http://localhost:54252/umbraco","SourceContext":"Umbraco.Core.Sync.ApplicationUrlHelper","ProcessId":13528,"ProcessName":"iisexpress","ThreadId":14,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":5,"HttpRequestId":"c111e35f-af5f-4a6d-be01-b700b947c7a4"}
{"@t":"2022-02-22T16:34:52.0260673Z","@mt":"ApplicationUrl: {UmbracoAppUrl} (UmbracoModule request)","UmbracoAppUrl":"http://localhost:54252/umbraco","SourceContext":"Umbraco.Core.Sync.ApplicationUrlHelper","ProcessId":13528,"ProcessName":"iisexpress","ThreadId":4,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":7,"HttpRequestId":"e675ec2c-06f8-41be-b688-af005b588487"}
{"@t":"2022-02-22T16:34:52.0260673Z","@mt":"ApplicationUrl: {UmbracoAppUrl} (UmbracoModule request)","UmbracoAppUrl":"http://localhost:54252/umbraco","SourceContext":"Umbraco.Core.Sync.ApplicationUrlHelper","ProcessId":13528,"ProcessName":"iisexpress","ThreadId":5,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":6,"HttpRequestId":"83933c23-aa49-46f2-aa48-6bc67d61082c"}
{"@t":"2022-02-22T16:34:53.6755167Z","@mt":"Registered with MainDom, localContentDbExists? {LocalContentDbExists}, localMediaDbExists? {LocalMediaDbExists}","LocalContentDbExists":true,"LocalMediaDbExists":true,"SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":13528,"ProcessName":"iisexpress","ThreadId":4,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":7,"HttpRequestId":"e675ec2c-06f8-41be-b688-af005b588487"}
{"@t":"2022-02-22T16:34:53.6835622Z","@mt":"Creating the content store, localContentDbExists? {LocalContentDbExists}","LocalContentDbExists":true,"SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":13528,"ProcessName":"iisexpress","ThreadId":4,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":7,"HttpRequestId":"e675ec2c-06f8-41be-b688-af005b588487"}
{"@t":"2022-02-22T16:34:53.6859470Z","@mt":"Creating the media store, localMediaDbExists? {LocalMediaDbExists}","LocalMediaDbExists":true,"SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":13528,"ProcessName":"iisexpress","ThreadId":4,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":7,"HttpRequestId":"e675ec2c-06f8-41be-b688-af005b588487"}
{"@t":"2022-02-22T16:34:53.7123977Z","@mt":"Forcing index {IndexerName} to be unlocked since it was left in a locked state","IndexerName":"InternalIndex","SourceContext":"Umbraco.Examine.ExamineExtensions","ProcessId":13528,"ProcessName":"iisexpress","ThreadId":3,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"b87d127d-4ef7-4e3a-acf3-d08898427910"}
{"@t":"2022-02-22T16:34:53.7156234Z","@mt":"Forcing index {IndexerName} to be unlocked since it was left in a locked state","IndexerName":"ExternalIndex","SourceContext":"Umbraco.Examine.ExamineExtensions","ProcessId":13528,"ProcessName":"iisexpress","ThreadId":3,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"b87d127d-4ef7-4e3a-acf3-d08898427910"}
{"@t":"2022-02-22T16:34:53.7187236Z","@mt":"Starting initialize async background thread.","SourceContext":"Umbraco.Web.Search.BackgroundIndexRebuilder","ProcessId":13528,"ProcessName":"iisexpress","ThreadId":3,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"b87d127d-4ef7-4e3a-acf3-d08898427910"}
{"@t":"2022-02-22T16:34:56.0662611Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Loading content from local cache file","TimingId":"cd365fd","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":13528,"ProcessName":"iisexpress","ThreadId":4,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":7,"HttpRequestId":"e675ec2c-06f8-41be-b688-af005b588487"}
{"@t":"2022-02-22T16:34:56.1927109Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Completed.","Duration":127,"TimingId":"cd365fd","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":13528,"ProcessName":"iisexpress","ThreadId":4,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":7,"HttpRequestId":"e675ec2c-06f8-41be-b688-af005b588487"}
{"@t":"2022-02-22T16:34:56.2300624Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Loading media from local cache file","TimingId":"6237a6c","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":13528,"ProcessName":"iisexpress","ThreadId":4,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":7,"HttpRequestId":"e675ec2c-06f8-41be-b688-af005b588487"}
{"@t":"2022-02-22T16:34:56.2330666Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Completed.","Duration":3,"TimingId":"6237a6c","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":13528,"ProcessName":"iisexpress","ThreadId":4,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"INFO ","HttpRequestNumber":7,"HttpRequestId":"e675ec2c-06f8-41be-b688-af005b588487"}
{"@t":"2022-02-22T16:48:33.7003479Z","@mt":"Exception ({InstanceId}).","@l":"Error","@x":"System.Data.SqlClient.SqlException (0x80131904): A transport-level error has occurred when receiving results from the server. (provider: TCP Provider, error: 0 - The semaphore timeout period has expired.) ---> System.ComponentModel.Win32Exception (0x80004005): The semaphore timeout period has expired\r\n   at System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)\r\n   at System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)\r\n   at System.Data.SqlClient.TdsParserStateObject.ReadSniError(TdsParserStateObject stateObj, UInt32 error)\r\n   at System.Data.SqlClient.TdsParserStateObject.ReadSniSyncOverAsync()\r\n   at System.Data.SqlClient.TdsParserStateObject.TryReadNetworkPacket()\r\n   at System.Data.SqlClient.TdsParserStateObject.TryPrepareBuffer()\r\n   at System.Data.SqlClient.TdsParserStateObject.TryReadByte(Byte& value)\r\n   at System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)\r\n   at System.Data.SqlClient.SqlDataReader.TryConsumeMetaData()\r\n   at System.Data.SqlClient.SqlDataReader.get_MetaData()\r\n   at System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString, Boolean isInternal, Boolean forDescribeParameterEncryption, Boolean shouldCacheForAlwaysEncrypted)\r\n   at System.Data.SqlClient.SqlCommand.RunExecuteReaderTds(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, Boolean async, Int32 timeout, Task& task, Boolean asyncWrite, Boolean inRetry, SqlDataReader ds, Boolean describeParameterEncryptionRequest)\r\n   at System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method, TaskCompletionSource`1 completion, Int32 timeout, Task& task, Boolean& usedCache, Boolean asyncWrite, Boolean inRetry)\r\n   at System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method)\r\n   at System.Data.SqlClient.SqlCommand.ExecuteReader(CommandBehavior behavior, String method)\r\n   at Umbraco.Core.Persistence.FaultHandling.RetryPolicy.ExecuteAction[TResult](Func`1 func) in D:\\a\\1\\s\\src\\Umbraco.Core\\Persistence\\FaultHandling\\RetryPolicy.cs:line 172\r\n   at NPoco.Database.ExecuteReaderHelper(DbCommand cmd)\r\n   at NPoco.Database.ExecuteDataReader(DbCommand cmd)\r\nClientConnectionId:8baee241-d2a2-46d7-a46e-7160b3452d39\r\nError Number:121,State:0,Class:20","InstanceId":"f1e51b45","SourceContext":"Umbraco.Core.Persistence.UmbracoDatabase","ProcessId":13528,"ProcessName":"iisexpress","ThreadId":91,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"ERROR"}
{"@t":"2022-02-22T16:48:35.4597782Z","@mt":"Failed (will repeat).","@l":"Error","@x":"System.Data.SqlClient.SqlException (0x80131904): Execution Timeout Expired.  The timeout period elapsed prior to completion of the operation or the server is not responding. ---> System.ComponentModel.Win32Exception (0x80004005): The wait operation timed out\r\n   at System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)\r\n   at System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)\r\n   at System.Data.SqlClient.TdsParserStateObject.ReadSniError(TdsParserStateObject stateObj, UInt32 error)\r\n   at System.Data.SqlClient.TdsParserStateObject.ReadSniSyncOverAsync()\r\n   at System.Data.SqlClient.TdsParserStateObject.TryReadNetworkPacket()\r\n   at System.Data.SqlClient.TdsParserStateObject.TryPrepareBuffer()\r\n   at System.Data.SqlClient.TdsParserStateObject.TryReadByte(Byte& value)\r\n   at System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)\r\n   at System.Data.SqlClient.TdsParser.Run(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj)\r\n   at System.Data.SqlClient.TdsParser.TdsExecuteTransactionManagerRequest(Byte[] buffer, TransactionManagerRequestType request, String transactionName, TransactionManagerIsolationLevel isoLevel, Int32 timeout, SqlInternalTransaction transaction, TdsParserStateObject stateObj, Boolean isDelegateControlRequest)\r\n   at System.Data.SqlClient.SqlInternalConnectionTds.ExecuteTransactionYukon(TransactionRequest transactionRequest, String transactionName, IsolationLevel iso, SqlInternalTransaction internalTransaction, Boolean isDelegateControlRequest)\r\n   at System.Data.SqlClient.SqlInternalConnection.BeginSqlTransaction(IsolationLevel iso, String transactionName, Boolean shouldReconnect)\r\n   at System.Data.SqlClient.SqlConnection.BeginTransaction(IsolationLevel iso, String transactionName)\r\n   at System.Data.SqlClient.SqlConnection.BeginDbTransaction(IsolationLevel isolationLevel)\r\n   at StackExchange.Profiling.Data.ProfiledDbConnection.BeginDbTransaction(IsolationLevel isolationLevel) in C:\\projects\\dotnet\\src\\MiniProfiler.Shared\\Data\\ProfiledDbConnection.cs:line 138\r\n   at NPoco.Database.BeginTransaction(IsolationLevel isolationLevel)\r\n   at Umbraco.Core.Scoping.Scope.get_Database() in D:\\a\\1\\s\\src\\Umbraco.Core\\Scoping\\Scope.cs:line 330\r\n   at Umbraco.Core.Sync.DatabaseServerMessenger.Sync() in D:\\a\\1\\s\\src\\Umbraco.Core\\Sync\\DatabaseServerMessenger.cs:line 263\r\n   at Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent.InstructionProcessTask.PerformRun() in D:\\a\\1\\s\\src\\Umbraco.Web\\Compose\\DatabaseServerRegistrarAndMessengerComponent.cs:line 163\r\nClientConnectionId:053322b0-f180-48fa-9190-1c8e4b8999b6\r\nError Number:-2,State:0,Class:11","SourceContext":"Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent+InstructionProcessTask","ProcessId":13528,"ProcessName":"iisexpress","ThreadId":72,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"ERROR"}
{"@t":"2022-02-22T16:48:55.4759778Z","@mt":"Failed (will repeat).","@l":"Error","@x":"System.InvalidOperationException: Timeout expired.  The timeout period elapsed prior to obtaining a connection from the pool.  This may have occurred because all pooled connections were in use and max pool size was reached.\r\n   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)\r\n   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.Open()\r\n   at NPoco.Database.OpenSharedConnectionImp(Boolean isInternal)\r\n   at NPoco.Database.BeginTransaction(IsolationLevel isolationLevel)\r\n   at Umbraco.Core.Scoping.Scope.get_Database() in D:\\a\\1\\s\\src\\Umbraco.Core\\Scoping\\Scope.cs:line 330\r\n   at Umbraco.Core.Sync.DatabaseServerMessenger.Sync() in D:\\a\\1\\s\\src\\Umbraco.Core\\Sync\\DatabaseServerMessenger.cs:line 263\r\n   at Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent.InstructionProcessTask.PerformRun() in D:\\a\\1\\s\\src\\Umbraco.Web\\Compose\\DatabaseServerRegistrarAndMessengerComponent.cs:line 163","SourceContext":"Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent+InstructionProcessTask","ProcessId":13528,"ProcessName":"iisexpress","ThreadId":58,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"ERROR"}
{"@t":"2022-02-22T16:49:42.4382360Z","@mt":"Failed to update server record in database.","@l":"Error","@x":"System.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server) ---> System.ComponentModel.Win32Exception (0x80004005): The semaphore timeout period has expired\r\n   at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)\r\n   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)\r\n   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)\r\n   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)\r\n   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.Open()\r\n   at Umbraco.Core.Persistence.FaultHandling.RetryPolicy.<>c__DisplayClass20_0.<ExecuteAction>b__0() in D:\\a\\1\\s\\src\\Umbraco.Core\\Persistence\\FaultHandling\\RetryPolicy.cs:line 125\r\n   at Umbraco.Core.Persistence.FaultHandling.RetryPolicy.ExecuteAction[TResult](Func`1 func) in D:\\a\\1\\s\\src\\Umbraco.Core\\Persistence\\FaultHandling\\RetryPolicy.cs:line 172\r\n   at NPoco.Database.AbortTransaction(Boolean fromComplete)\r\n   at Umbraco.Core.Scoping.Scope.DisposeLastScope() in D:\\a\\1\\s\\src\\Umbraco.Core\\Scoping\\Scope.cs:line 621\r\n   at Umbraco.Core.Scoping.Scope.Dispose() in D:\\a\\1\\s\\src\\Umbraco.Core\\Scoping\\Scope.cs:line 557\r\n   at Umbraco.Core.Services.Implement.ServerRegistrationService.TouchServer(String serverAddress, String serverIdentity, TimeSpan staleTimeout) in D:\\a\\1\\s\\src\\Umbraco.Core\\Services\\Implement\\ServerRegistrationService.cs:line 89\r\n   at Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent.TouchServerTask.PerformRun() in D:\\a\\1\\s\\src\\Umbraco.Web\\Compose\\DatabaseServerRegistrarAndMessengerComponent.cs:line 206\r\nClientConnectionId:00000000-0000-0000-0000-000000000000\r\nError Number:121,State:0,Class:20","SourceContext":"Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent","ProcessId":13528,"ProcessName":"iisexpress","ThreadId":91,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"ERROR"}
{"@t":"2022-02-22T16:49:42.4420731Z","@mt":"Failed (will repeat).","@l":"Error","@x":"System.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server) ---> System.ComponentModel.Win32Exception (0x80004005): The semaphore timeout period has expired\r\n   at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)\r\n   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)\r\n   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)\r\n   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)\r\n   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.Open()\r\n   at NPoco.Database.OpenSharedConnectionImp(Boolean isInternal)\r\n   at NPoco.Database.BeginTransaction(IsolationLevel isolationLevel)\r\n   at Umbraco.Core.Scoping.Scope.get_Database() in D:\\a\\1\\s\\src\\Umbraco.Core\\Scoping\\Scope.cs:line 330\r\n   at Umbraco.Core.Sync.DatabaseServerMessenger.Sync() in D:\\a\\1\\s\\src\\Umbraco.Core\\Sync\\DatabaseServerMessenger.cs:line 263\r\n   at Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent.InstructionProcessTask.PerformRun() in D:\\a\\1\\s\\src\\Umbraco.Web\\Compose\\DatabaseServerRegistrarAndMessengerComponent.cs:line 163\r\nClientConnectionId:00000000-0000-0000-0000-000000000000\r\nError Number:121,State:0,Class:20","SourceContext":"Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent+InstructionProcessTask","ProcessId":13528,"ProcessName":"iisexpress","ThreadId":72,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"ERROR"}
{"@t":"2022-02-22T16:50:19.2231776Z","@mt":"Failed (will repeat).","@l":"Error","@x":"System.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server) ---> System.ComponentModel.Win32Exception (0x80004005): The network path was not found\r\n   at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)\r\n   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)\r\n   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)\r\n   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)\r\n   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.Open()\r\n   at NPoco.Database.OpenSharedConnectionImp(Boolean isInternal)\r\n   at NPoco.Database.BeginTransaction(IsolationLevel isolationLevel)\r\n   at Umbraco.Core.Scoping.Scope.get_Database() in D:\\a\\1\\s\\src\\Umbraco.Core\\Scoping\\Scope.cs:line 330\r\n   at Umbraco.Core.Sync.DatabaseServerMessenger.Sync() in D:\\a\\1\\s\\src\\Umbraco.Core\\Sync\\DatabaseServerMessenger.cs:line 263\r\n   at Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent.InstructionProcessTask.PerformRun() in D:\\a\\1\\s\\src\\Umbraco.Web\\Compose\\DatabaseServerRegistrarAndMessengerComponent.cs:line 163\r\nClientConnectionId:00000000-0000-0000-0000-000000000000\r\nError Number:53,State:0,Class:20","SourceContext":"Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent+InstructionProcessTask","ProcessId":13528,"ProcessName":"iisexpress","ThreadId":101,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"ERROR"}
{"@t":"2022-02-22T16:50:39.6091539Z","@mt":"Failed (will repeat).","@l":"Error","@x":"System.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server) ---> System.ComponentModel.Win32Exception (0x80004005): The network path was not found\r\n   at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)\r\n   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)\r\n   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)\r\n   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)\r\n   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.Open()\r\n   at NPoco.Database.OpenSharedConnectionImp(Boolean isInternal)\r\n   at NPoco.Database.BeginTransaction(IsolationLevel isolationLevel)\r\n   at Umbraco.Core.Scoping.Scope.get_Database() in D:\\a\\1\\s\\src\\Umbraco.Core\\Scoping\\Scope.cs:line 330\r\n   at Umbraco.Core.Sync.DatabaseServerMessenger.Sync() in D:\\a\\1\\s\\src\\Umbraco.Core\\Sync\\DatabaseServerMessenger.cs:line 263\r\n   at Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent.InstructionProcessTask.PerformRun() in D:\\a\\1\\s\\src\\Umbraco.Web\\Compose\\DatabaseServerRegistrarAndMessengerComponent.cs:line 163\r\nClientConnectionId:00000000-0000-0000-0000-000000000000\r\nError Number:53,State:0,Class:20","SourceContext":"Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent+InstructionProcessTask","ProcessId":13528,"ProcessName":"iisexpress","ThreadId":98,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"ERROR"}
{"@t":"2022-02-22T16:50:51.5685731Z","@mt":"Failed (will repeat).","@l":"Error","@x":"System.Data.SqlClient.SqlException (0x80131904): A connection was successfully established with the server, but then an error occurred during the login process. (provider: SSL Provider, error: 0 - The network connection was aborted by the local system.) ---> System.ComponentModel.Win32Exception (0x80004005): The network connection was aborted by the local system\r\n   at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)\r\n   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)\r\n   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)\r\n   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)\r\n   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)\r\n   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.Open()\r\n   at NPoco.Database.OpenSharedConnectionImp(Boolean isInternal)\r\n   at NPoco.Database.BeginTransaction(IsolationLevel isolationLevel)\r\n   at Umbraco.Core.Scoping.Scope.get_Database() in D:\\a\\1\\s\\src\\Umbraco.Core\\Scoping\\Scope.cs:line 330\r\n   at Umbraco.Core.Sync.DatabaseServerMessenger.Sync() in D:\\a\\1\\s\\src\\Umbraco.Core\\Sync\\DatabaseServerMessenger.cs:line 263\r\n   at Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent.InstructionProcessTask.PerformRun() in D:\\a\\1\\s\\src\\Umbraco.Web\\Compose\\DatabaseServerRegistrarAndMessengerComponent.cs:line 163\r\nClientConnectionId:657cf495-b88c-4353-9721-826e2e5e7f1f\r\nError Number:1236,State:0,Class:20","SourceContext":"Umbraco.Web.Compose.DatabaseServerRegistrarAndMessengerComponent+InstructionProcessTask","ProcessId":13528,"ProcessName":"iisexpress","ThreadId":98,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"AAR00118","Log4NetLevel":"ERROR"}
